cpp/ql/src/experimental/Best Practices/UselessTest.ql

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Looks like the file needs to be autoformatted 😄.

Reply:
Should be better now